libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -69,60 +67,47 @@ public static String getIpvSessionId(JourneyRequest event)
         return getIpvSessionId(event, false);
     }
 
-    public static String getIpvSessionIdAllowBlank(JourneyRequest event)
+    public static String getIpvSessionIdAllowMissing(JourneyRequest event)
             throws HttpResponseExceptionWithErrorBody {
         return getIpvSessionId(event, true);
     }
 
-    public static String getIpAddress(APIGatewayProxyRequestEvent event)
-            throws HttpResponseExceptionWithErrorBody {
+    public static String getIpAddress(APIGatewayProxyRequestEvent event) {
         return getIpAddress(event.getHeaders());
     }
 
     public static String getEncodedDeviceInformation(APIGatewayProxyRequestEvent event) {
         return RequestHelper.getHeaderByKey(event.getHeaders(), ENCODED_DEVICE_INFORMATION_HEADER);
     }
 
-    public static String getIpAddress(JourneyRequest request)
-            throws HttpResponseExceptionWithErrorBody {
-        String ipAddress = request.getIpAddress();
-        validateIpAddress(ipAddress, "ipAddress not present in request.");
+    public static String getIpAddress(JourneyRequest request) {
+        var ipAddress = nullIfBlank(request.getIpAddress());
+        validateIpAddress(ipAddress);
         return ipAddress;
     }
 
-    public static String getLanguage(CriJourneyRequest request)
-            throws HttpResponseExceptionWithErrorBody {
-        String language = request.getLanguage();
+    public static String getLanguage(CriJourneyRequest request) {
+        var language = nullIfBlank(request.getLanguage());
 
         if (language == null) {
-            LOGGER.error(
-                    LogHelper.buildErrorMessage(
-                            "Language choice is missing from this frontend request",
-                            IP_ADDRESS_HEADER));
-            throw new HttpResponseExceptionWithErrorBody(
-                    SC_BAD_REQUEST, ErrorResponse.MISSING_LANGUAGE);
+            LOGGER.warn(LogHelper.buildErrorMessage(ErrorResponse.MISSING_LANGUAGE));
```

Comment:
I'd be willing the change this back, but IMO a missing language header should not be fatal - especially as it's only required for DWP KBVs.